README.md

```diff
@@ -1,79 +1,97 @@
-# Installation
-## Docker
-We're providing a Docker image for this project. Head to [Installing with Docker](https://ostis-ai.github.io/sc-machine/docker) 
-section of our docs to try it out! We recommend to use this way for sc-machine deployment.
-## Native
-If you do not have the opportunity to deploy the system using docker, please, refer to the docs for your OS: 
-[Build on Linux](https://ostis-ai.github.io/sc-machine/build/linux-build/) or [Build on macOS](https://ostis-ai.github.io/sc-machine/build/osx-build/).
-Nowadays sc-machine isn't *natively* support on Windows.
-
-# Usage
-Typical usage case is to install platform using Docker or by building from sources, build knowledge base and start 
-**sc-server** for other apps. By default, sc-server is runnable on host 127.0.0.1 and port 8090. Before sc-server 
-running, sure, that all needed SCs and SCg sources is located in kb folder.
-
-To run sc-server, you can do the following:
-```sh
-#build kb
-python3 scripts/build_kb.py -c config/sc-machine.ini <path to kb folder with SCs and SCg sources or repo.path file>
-#launch server
-./bin/sc-server -c config/sc-machine.ini
-``` 
-
-You can use `healthcheck.py` script to check memory state in sc-server.
-```sh
-python3 scripts/healthcheck.py --host <sc-server host> --port <sc-server port>
-``` 
-
-To get more information about these scripts use `-h` flags upon their launch.
+Short version: **sc-machine** is an agent-based knowledge graph processing machine.
+
+The knowledge graphs are represented in SC-code (language of universal knowledge representation) provided by the [**OSTIS Technology**](https://github.com/ostis-ai/ostis-project).
+
+sc-machine allows you to integrate problem solutions from different subject areas **by using the same**:
+
+- **technology**
+- **programming and data representation language**
+- **software stack**.
+
+This project contains:
+
+- `sc-memory` - semantic storage to store graph constructions
+- `sc-event-manager` event-based processing (async program workflow handling)
+- tools to communicate with `sc-memory`:
+  - native C++ API
+  - `sc-server` - a Websocket server that provides a network API to communicate with storage using JSON
+  - `sc-builder` loads information from SCs files into storage (SCs-code is a one of external representation languages of SC-code)
+
+<details>
+   <summary>More info</summary>
+
+sc-machine is a **platform-independent graph database management system** that can store / retrieve knowledge graphs and run tasks (agents) on them.
+
+Both declarative (data, data structures, documentation, tasks specification, etc.) and procedural
+(programs, modules, systems, communication between systems) knowledge is represented using the same language: SC-code.
+
+</details>
+
+## Documentation
+
+- A brief user manual and developer docs are hosted on our [GitHub Pages](https://ostis-ai.github.io/sc-machine).
+- Full documentation, including:
+  - core concepts
+  - rationale behind sc-machine
```

Comment:
```suggestion
  - components implementation
  - sc-model architecture and sc-memory model
```

Reply:
I haven't heard about `sc-model` earlier. 

Anyway, this suggestion and the one below can be generalised to a single one - replacing more abstract terms understood by the industry with ones specific to our project. But I do not really agree with that. The complexity of the terms used in the README was pretty much the reason why I've made the change - this document was too hard to be understood by a person which has never interacted with OSTIS before.
Basically, `system design` is `SC model architecture + sc-memory model + implementation details`, so why wouldn't we want to use a more eloquent term to express it?

Reply:
I would like to express myself in terms and concepts interpreted within the framework of OSTIS technology. Documentation must be represented in these terms and concepts

Reply:
After discuss: leave Pavel's version before approving core concepts of SC-machine